source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -3295,6 +3295,21 @@
 
 /*---------------------------------------------------------------------------*/
 
+/*
+ * ipconfigMAC_FILTERING
+ *
+ * Type: BaseType_t ( ipconfigENABLE | ipconfigDISABLE )
+ *
+ * When set to ipconfigENABLE, this macro will adds two functions
+ * to the network interface. Those functions allow adding/removing multicast
+ * MAC addresses to/from the hardware EMAC. It is up to the portable network
```

Comment:
~~~diff
-    this macro will adds two functions
+    this macro will add two functions
~~~

~~~diff
-    adding/removing multicast MAC addresses
+    adding/removing MAC addresses
~~~